lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1399,69 +1413,61 @@ export class UserAgentApplication {
   }
 
   /**
-   * Creates a requestInfo object from the URL fragment and returns it.
+   * Returns deserialized portion of URL hash
+   * @param hash
+   */
+  private stringifyHash(hash: string) {
```

Comment:
name change.. it's not really stringifying. It seems to build a NVP object

Reply:
renamed function to deserializeHash